Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Controller.purs

```diff
@@ -1257,37 +1257,30 @@ eval (UpdateCurrentStage stage (RideBookingRes resp)) state = do
-      "FindingQuotes" -> do
-        let secondsLeft = findingQuotesSearchExpired false true
-        case (methodArgumentCount "startLottieProcess") == 1 of
-          true  -> do
-            let findingQuotesProgress = 1.0 - (toNumber secondsLeft)/(toNumber (getSearchExpiryTime true))
-            if secondsLeft > 0 then
-              void $ pure $ startLottieProcess lottieAnimationConfig {rawJson = "progress_loader_line", lottieId = (getNewIDWithTag "lottieLoaderAnimProgress"), minProgress = findingQuotesProgress, scaleType="CENTER_CROP"}
-            else pure unit
-          false -> pure unit
-        case flowWithoutOffers WithoutOffers of
-          true | secondsLeft <= 0 -> do
-              _ <- pure $ updateLocalStage QuoteList
-              continueWithCmd state [do
-                let response = SelectListRes { selectedQuotes: Nothing, bookingId : Nothing }
-                pure $ GetQuotesList response
-              ]
-          _ -> continue state
-      "QuoteList" -> do
-        let findingQuotesProgress = 1.0 - 30.0/(toNumber (getSearchExpiryTime true))
-        void $ pure $ startLottieProcess lottieAnimationConfig {rawJson = "progress_loader_line", lottieId = (getNewIDWithTag "lottieLoaderAnimProgress"), minProgress = findingQuotesProgress, scaleType="CENTER_CROP"}
-        continue state
-      "RideAccepted" | (state.data.fareProductType == FPT.ONE_WAY_SPECIAL_ZONE) -> exit $ Retry state
-      "ConfirmingQuotes" -> do
-        let findingQuotesProgress = 1.0 - 30.0/(toNumber (getSearchExpiryTime false))
-        void $ pure $ startLottieProcess lottieAnimationConfig {rawJson = "progress_loader_line", lottieId = (getNewIDWithTag "lottieLoaderAnimProgress"), minProgress = findingQuotesProgress, scaleType="CENTER_CROP"}
-        continue state
-      _ -> continue state
+  case getValueToLocalNativeStore LOCAL_STAGE of
+    "FindingQuotes" -> do
+      let secondsLeft = findingQuotesSearchExpired false
+      case (methodArgumentCount "startLottieProcess") == 1 of
+        true  -> do
+          let findingQuotesProgress = 1.0 - (toNumber secondsLeft)/(toNumber (getSearchExpiryTime "LazyCheck"))
+          if secondsLeft > 0 then
+            void $ pure $ startLottieProcess lottieAnimationConfig {rawJson = (getAssetsBaseUrl FunctionCall) <> "lottie/progress_loader_line.json", lottieId = (getNewIDWithTag "lottieLoaderAnimProgress"), minProgress = findingQuotesProgress, scaleType="CENTER_CROP"}
+          else pure unit
+        false -> pure unit
+      case flowWithoutOffers WithoutOffers of
+        true | secondsLeft <= 0 -> do
+            _ <- pure $ updateLocalStage QuoteList
+            continueWithCmd state [do
+              let response = SelectListRes { selectedQuotes: Nothing, bookingId : Nothing }
+              pure $ GetQuotesList response
+            ]
+        _ -> continue state
+    "QuoteList" -> do
+      let findingQuotesProgress = 1.0 - 30.0/(toNumber (getSearchExpiryTime "LazyCheck"))
```

Comment:
@aryan010803 did you check this in case killing and reopening because there are other cases as well in which we show this line and that might be dependent on the fare Product Type.

Reply:
@MercyQueen  Yup its working fine. when we kill the app and reopen it again the loader line doesn't starts from 0